### ISSUE TITLE
Source IP blocked by cloudflare

### ISSUE THREAD

**Issue by codium9** [KotatsuApp/kotatsu-parsers#1046]
### Source information

KotatsuApp

### Steps to reproduce

Result : Unable to pass capcha to access manga

### Kotatsu version

7.5.2

### Android version

14

### Other details

Source [FR] : EpsilonScan 
URL site : epsilonscan.to

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by codium9** [comment]
Duplicate #1038
